src/web/data/userMapData.json

```diff
@@ -61,9 +64,11 @@
       "Suzanne Cory High School",
       "Swinburne",
       "Swinburne Online",
+      "Swinburne Business School",
```

Comment:
Have you sorted the entries? Because alphabetically this would come before `Swinburne Online`.

Reply:
Done fixed it